### DISCUSSION TITLE
Setting the max length of strings within an array?

### DISCUSSION THREAD

**Discussion by SheaBelsky** [prisma/prisma#14103] [Q&A]
Consider this Prisma schema:

```
model User {
  tags  String[]
  bio   String?  @db.VarChar(5000)
}
```

If I want to constrain the length of an individual string within the `tags` field, how would I do that? I'm aware that I can constrain the length of a single string with `@db.VarChar(5000)`, but is that possible with an array type?

**Comment by austincrim** [comment]
Hey @SheaBelsky 👋,

I just tested this with Postgres and it seems that it is possible 👍.

```prisma
model User {
  tags  String[] @db.VarChar(50)
  bio   String?  @db.VarChar(5000)
}
```

It might vary depending on your database provider, though.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
